src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,309 @@
+  --pf-c-sequence__link--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-sequence__link--BorderWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-sequence__link--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--hover--Transform: scale(1.333333);
+  --pf-c-sequence__link--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--hover--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--active--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--active--Transform: scale(1.333333);
+  --pf-c-sequence__link--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--active--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--focus--Transform: scale(1.333333);
+  --pf-c-sequence__link--focus--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--focus--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--m-current--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--m-current--Transform: scale(1.333333);
+  --pf-c-sequence__link--m-current--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--m-current--BorderColor: var(--pf-global--primary-color--100);
+
+  // Content
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__content--MaxWidth: #{pf-size-prem(140px)};
+
+  // Content Title Color
+  --pf-c-sequence__content-title--Color: var(--pf-global--Color--100);
+  --pf-c-sequence__content-title--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--focus--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--m-current--Color: var(--pf-global--primary-color--100);
+
+  // Content Title Font
+  --pf-c-sequence__content-title--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-sequence__content-title--FontWeight: var(--pf-global--FontWeight--reguar);
+  --pf-c-sequence__content-title--active--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--focus--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--m-current--FontWeight: var(--pf-global--FontWeight--semi-bold);
+
+
+  // Content Status Color
+  --pf-c-sequence__content-status--Color: var(--pf-global--Color--200);
+
+  // Content Status Font
+  --pf-c-sequence__content-status--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-sequence__content-status--FontWeight: var(--pf-global--FontWeight--regular);
+
+  display: flex;
+  justify-content: center;
+  height: var(--pf-c-sequence--Height);
+  overflow-y: hidden;
+
+  // remove default border styling
+  button {
```

Comment:
I think all of the CSS here should be under `.pf-c-sequence__control`?

Reply:
When I move it to sequence__control those button styles dont show, so moved it back to the sequence

Reply:
Hmm, I tested this on my end and it seems to work, but maybe I'm missing something? I created a PR if you want to check it out - https://github.com/christiemolloy/patternfly-next/pull/4